app/konnect-platform/teams-and-roles.md

```diff
@@ -0,0 +1,268 @@
+  - team: Organization Admin (Read Only)
+    description: Users can view all entities and configuration in the organization.
+  - team: Portal Admin
+    description: Users can fully manage all Dev Portal content, which includes {{site.konnect_short_name}} service pages and supporting content, as well as Dev Portal configuration and Service connections. <br> To manage app registration requests, members must also be assigned to the Admin or Maintainer roles for the corresponding Services.
+  - team: API Product Admin
+    description: Users can create and manage API products, including publishing API product versions to Dev Portal and enabling application registration.
+  - team: API Product Developer
+    description: Users can create and manage versions of API products.
+  - team: Control Plane Admin
+    description: Users can create and manage Control Planes.
+{% endtable %} 
+
+## Roles
+
+Roles predefine access to a particular resource, or
+instances of a particular resource type (for example, API product roles can be scoped to a particular API product or all API products while Control Plane roles can be scoped to a particular Control Plane or all Control Planes).
+
+You can manage a user's roles by navigating to [**Organization**](https://cloud.konghq.com/organization/) > **Users** in {{site.konnect_short_name}} and clicking the **Role Assignments** tab for a user.
+
+### Predefined roles
+
+{{site.konnect_short_name}} provides the following predefined roles.
+
+#### API Products
+
+The following describes the predefined roles for API Products:
+
+<!-- vale off -->
+{% konnect_roles_table %}
+schema: api_products
+{% endkonnect_roles_table %}
+<!-- vale on -->
+
+#### Control Planes
+
+The following describes the predefined roles for Control Planes:
+
+<!-- vale off -->
+{% konnect_roles_table %}
+schema: control_planes
+{% endkonnect_roles_table %}
+<!-- vale on -->
+
+#### Audit logs
+
+The following describes the predefined roles for audit logs:
+
+<!-- vale off -->
+{% konnect_roles_table %}
+schema: audit_logs
+{% endkonnect_roles_table %}
+<!-- vale on -->
+
+#### Identity
+
+The following describes the predefined roles for identity:
+
+<!-- vale off -->
+{% konnect_roles_table %}
+schema: identity
+{% endkonnect_roles_table %}
+<!-- vale on -->
+
+#### Mesh control planes
+
+The following describes the predefined roles for Mesh:
+
+<!-- vale off -->
+{% konnect_roles_table %}
+schema: mesh_control_planes
+{% endkonnect_roles_table %}
+<!-- vale on -->
+
+#### Networks 
+
+The following describes the predefined roles for networks:
+
+{% table %}
+columns:
+  - title: Role
+    key: role
+  - title: Description
+    key: description
+rows:
+  - role: Network Admin 
+    description: Access to all read and write permissions related to a network.
+  - role: Network Creator
+    description: Access to creating networks.
+  - role: Network Viewer
+    description: Access to read-only permissions to networks.
```

Comment:
```suggestion
rows:
  - role: "`Network Admin`"
    description: Access to all read and write permissions related to a network.
  - role: "`Network Creator`"
    description: Access to creating networks.
  - role: "`Network Viewer`"
    description: Access to read-only permissions to networks.
```
In the tables generated from the schemas, the role is in code-formatted, so this would make it consistent.